app-ios/Modules/Sources/Timetable/TimetableListItemView.swift

```diff
@@ -5,17 +5,17 @@ import Theme
 
 struct TimetableListItemView: View {
     let timetableItemWithFavorite: TimetableItemWithFavorite
-
+    
```

Comment:
I think this causes SwiftLint warning 👀 
Please check this 🙏🏼

Reply:
Ah, that's true. I will fix it 🙏
By the way, my Xcode arranges my code like this line by using `control + i` command ☹️
Is there any good way to avoid this incorrect formatting by Xcode?

Reply:
> Is there any good way to avoid this incorrect formatting by Xcode?

This setting is useful! `Automatically trim trailing whitespace`

<img width="841" alt="スクリーンショット 2023-08-23 2 59 45" src="https://github.com/DroidKaigi/conference-app-2023/assets/30540303/9b159a93-ee93-4334-95de-a9295bd5888b">

Reply:
Thank you! Reformatted! https://github.com/DroidKaigi/conference-app-2023/pull/852/commits/7012ea6c90cced19c569c982ba45afe8eb8085db